### PR TITLE
Adding new model to the Hub

### PR DESCRIPTION
**Pull request by hahuyhoang411**
- [x] Stealth v1.1: New model optimized on Math and Logical
- [x] Magicoder-S-DS-6.7B: Top coding model https://github.com/janhq/jan/issues/1177
- [x] CodeNinja-1.0-OpenChat-7b: Top coding model https://github.com/janhq/jan/issues/1182

### CODE REVIEW

**Comment by MrBenzWorld** [comment]
Thank you,  all these models are great for code generations great.

**Comment by louis-jan** [comment]
@hahuyhoang411 please help bump the model-extension version

File: models/stealth-v1.1-7b/model.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "metadata": {
+      "author": "Jan",
+      "tags": [
+        "7B",
+        "Merged"
+      ],
+      "size": 4370000000
+    },
+    "engine": "nitro",
+    "state": "ready"
```
Comment (louis-jan) [inline models/stealth-v1.1-7b/model.json:25]: remove state

File: models/codeninja-1.0-7b/model.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "object": "model",
+    "name": "CodeNinja 7B Q4",
+    "version": "1.0",
+    "description": "CodeNinja is an enhanced version of the renowned model openchat/openchat-3.5-1210. It is designed to be an indispensable tool for coders",
+    "format": "gguf",
+    "settings": {
+      "ctx_len": 4096,
+      "prompt_template": "GPT4 Correct User: {prompt}<|end_of_turn|>GPT4 Correct Assistant:"
+    },
+    "parameters": {
```
Comment (0xSage) [inline models/codeninja-1.0-7b/model.json:13]: Going forward, you don't have to always set parameters and most of the properties. It should default to a global config in the future.

**Review by 0xSage — APPROVED** [review]

File: models/codeninja-1.0-7b/model.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "source_url": "https://huggingface.co/beowolx/CodeNinja-1.0-OpenChat-7B-GGUF/resolve/main/codeninja-1.0-openchat-7b.Q4_K_M.gguf",
+    "id": "codeninja-1.0-7b",
+    "object": "model",
+    "name": "CodeNinja 7B Q4",
+    "version": "1.0",
+    "description": "CodeNinja is an enhanced version of the renowned model openchat/openchat-3.5-1210. It is designed to be an indispensable tool for coders",
```
Comment (0xSage) [inline models/codeninja-1.0-7b/model.json:7]: Let's be a bit more concise and pragmatic in our descriptions. No hyperbolic statements. 

Just "description": "CodeNinja is finetuned on openchat/openchat-3.5-1210. Good for coding.",

**Review by 0xSage — APPROVED** [review]

**Review by louis-jan — APPROVED** [review]
LGTM

**Pull request merged**